src/App.tsx

```diff
@@ -48,6 +47,19 @@ const App = () => {
     return !oriaParameterIsSet() ? <Header /> : null;
   }
 
+  function getIsbnFromQueryOrPath() {
+    const searchQuery = new URLSearchParams(window.location.search);
+    let isbn = searchQuery.get('isbn');
+    if (!isbn) {
+      let path = window.location.pathname;
+      if (path.substring(path.length - 1) === '/') {
+        path = path.substring(0, path.length - 1);
+      }
+      isbn = path.substring(path.lastIndexOf('/') + 1);
+    }
```

Comment:
Er det en grunn til å ikke kjøre isbn = window.location.pathname.split('/').slice(-1).pop()?

Reply:
Linje 55-57 er for å fjerne evt. "/" på slutten av path, og linje 58 fungerer på samme måte som slice-pop-metoden, men typescript klager på den (sikkert pga det du sier).